vyperlang/vyper
# Remove loaded limits for int128 and address

### What I did
Fix #2453 

### How I did it

### How to verify it
Check tests. Also check out where variables start from, reduces gas usage somewhat.

```python
@external
def foo():
    x: uint256 = 1
```
before
![Screenshot from 2021-10-22 12-40-13](https://user-images.githubusercontent.com/3867501/138492011-7b56962c-7a0a-4b4e-9c22-491ea7743f57.png)

after
![Screenshot from 2021-10-22 12-40-03](https://user-images.githubusercontent.com/3867501/138492002-5273a71a-2971-46d8-b8ab-c86137e54ffc.png)



### Description for the changelog

### Cute Animal Picture

![Put a link to a cute animal picture inside the parenthesis-->]()
